tests/components/homekit/util.py

```diff
@@ -0,0 +1,34 @@
+"""Test util for the homekit integration."""
+
+from asynctest import patch
+
+from homeassistant.components.homekit.const import DOMAIN
+from homeassistant.const import CONF_NAME, CONF_PORT
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+PATH_HOMEKIT = "homeassistant.components.homekit"
+
+
+async def async_init_integration(hass: HomeAssistant) -> MockConfigEntry:
+    """Set up the homekit integration in Home Assistant."""
+
+    with patch(f"{PATH_HOMEKIT}.HomeKit.async_start"):
+        entry = MockConfigEntry(
+            domain=DOMAIN, data={CONF_NAME: "mock_name", CONF_PORT: 12345}
+        )
+        entry.add_to_hass(hass)
+        assert await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
+        return entry
+
+
+async def async_init_entry(hass: HomeAssistant, entry: MockConfigEntry):
+    """Set up the homekit integration in Home Assistant."""
+
+    with patch(f"{PATH_HOMEKIT}.HomeKit.async_start"):
+        entry.add_to_hass(hass)
+        assert await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5453535)